_includes/cockroachcloud/sql-connection-string.md

```diff
@@ -17,4 +17,11 @@
     ~~~ shell
     cockroach sql --url "postgresql://<user>@<cluster-name>-<short-id>.<region>.<host>:26257/<database>?sslmode=verify-full&sslrootcert=$env:appdata\CockroachCloud\certs\$<cluster-name>-ca.crt"
     ~~~
-    </section>
\ No newline at end of file
+    </section>
+    
+    Where:
```

Comment:
Shouldn't this define the host, region, and database variables here as well?

Reply:
looks like host is always `cockroachlabs.cloud`, so I'll update that and add the other two variables